com.core447.StreamController.yml

```diff
@@ -0,0 +1,105 @@
+id: com.core447.StreamController
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: launch.sh
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=xdg-documents
+  - --filesystem=home
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=session-bus # Allow plugins to connect to the session bus
+  - --device=all # Needed to communicate with the decks
+  - --talk-name=org.freedesktop.Flatpak # Allow plugins to run system commands
+
+modules:
+  - pypi-requirements.yaml
+
+  - name: StreamController
```

Comment:
since this is the MAIN it should be last in the list. It's logical.

Reply:
ok